resources/META-INF/plugin.xml

```diff
@@ -16,7 +16,7 @@
   <version>1.15.1</version>
   <vendor>Twitter, Inc.</vendor>
 
-  <idea-version since-build="201.6073" until-build="201.*"/>
+  <idea-version since-build="201.6073" until-build="202.*"/>
```

Comment:
just checking that there's no breaking API changes from 201.x right? otherwise we'll need to be bump `since-build`?

Reply:
the `OpenProjectTask` signature has changed, so indeed, it would be safer to bump the `since-build` field

Reply:
ty!